frontend/app/src/pages/main/workflow-runs/$run/components/step-run-logs.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  LogLineOrderByDirection,
+  StepRun,
+  StepRunStatus,
+  queries,
+} from '@/lib/api';
+import { useQuery } from '@tanstack/react-query';
+import { CodeEditor } from '@/components/ui/code-editor';
+
+export function StepRunLogs({ stepRun }: { stepRun: StepRun | undefined }) {
+  const getLogsQuery = useQuery({
+    ...queries.stepRuns.getLogs(stepRun?.metadata.id || '', {
+      orderByDirection: LogLineOrderByDirection.Asc,
+    }),
+    enabled: !!stepRun,
+    refetchInterval: () => {
+      if (stepRun?.status === StepRunStatus.RUNNING) {
+        return 1000;
+      }
+
+      return 5000;
+    },
+  });
+
+  const logLines =
+    getLogsQuery.data?.rows?.reduce(
+      (acc, row) => acc + row.createdAt + ': ' + row.message + '\n',
+      '',
+    ) || 'No logs found';
```

Comment:
once written maybe we should link to the docs for logs if the step ran with no logs

Reply:
Yes